lte/cloud/go/services/lte/servicers/protected/builder_servicer.go

```diff
@@ -467,6 +470,54 @@ func getEnodebConfigsBySerial(nwConfig *lte_models.NetworkCellularConfigs, gwCon
 	return ret
 }
 
+func getNeighborCell(enbConfig *lte_models.EnodebConfig) map[string]*lte_mconfig.EnodebD_EnodebConfigNeighborCellTable {
+	if enbConfig.ConfigType == "MANAGED" {
+		cellularNeighborCell := enbConfig.ManagedConfig.NeighborCellList
+		ret := make(map[string]*lte_mconfig.EnodebD_EnodebConfigNeighborCellTable, len(cellularNeighborCell))
+		for _, neighbor := range cellularNeighborCell {
+			enbNeighborCell := &lte_mconfig.EnodebD_EnodebConfigNeighborCellTable{}
+			enbNeighborCell.Index = neighbor.Index
+			enbNeighborCell.CellId = neighbor.CellID
+			enbNeighborCell.Earfcn = neighbor.Earfcn
+			enbNeighborCell.Pci = neighbor.Pci
+			enbNeighborCell.Tac = neighbor.Tac
+			enbNeighborCell.QOffset = neighbor.QOffset
+			enbNeighborCell.Cio = neighbor.Cio
+			enbNeighborCell.Enable = neighbor.Enable
+			enbNeighborCell.Plmn = neighbor.Plmn
+			index := strconv.Itoa(int(neighbor.Index))
+			ret[index] = enbNeighborCell
+		}
+		return ret
+	}
+	return nil
```

Comment:
This should return an error if the config type is unmanaged - then handle that error in line 430/431

Reply:
This had a  condition in line 419/420, when the condition is true, the config type must be mangaged - so it would not need to modify.

Reply:
Already delete the if condition .

Reply:
This had a condition in line 419/420, when the condition is true, the config type must be mangaged - so already delete the if condition.